src/Fixer/Operator/BinaryOperatorSpacesFixer.php

```diff
@@ -135,11 +279,20 @@ protected function applyFix(\SplFileInfo $file, Tokens $tokens)
     protected function createConfigurationDefinition()
     {
         return new FixerConfigurationResolver([
-            (new FixerOptionBuilder('align_double_arrow', 'Whether to apply, remove or ignore double arrows alignment.'))
+            (new FixerOptionBuilder('default', 'default fix strategy.'))
+                ->setDefault(self::SINGLE_SPACE)
+                ->setAllowedValues(self::$allowedValues)
+                ->getOption(),
+            (new FixerOptionBuilder('operators', 'List of binary operator to fix.'))
```

Comment:
description is wrong.
we need to say that this is assoc key with key being ... and value being ...